examples/blog/src/pages/index.astro

```diff
@@ -1,62 +1,45 @@
 ---
 import BaseHead from '../components/BaseHead.astro';
 import Header from '../components/Header.astro';
-import BlogPostPreview from '../components/BlogPostPreview.astro';
-
-let title = 'Example Blog';
-let description = 'The perfect starter for your perfect blog.';
-
-// Use Astro.glob to fetch all post with associated frontmatter
-const unsortedPosts = await Astro.glob('./posts/*.md');
-const posts = unsortedPosts.sort(function (a, b) {
-	return (
-		new Date(b.frontmatter.publishDate).valueOf() - new Date(a.frontmatter.publishDate).valueOf()
-	);
-});
+import Footer from '../components/Footer.astro';
+import { SITE_TITLE, SITE_DESCRIPTION } from '../config.ts';
 ---
 
-<html lang="en">
+<!DOCTYPE html>
+<html lang="en-us">
 	<head>
-		<BaseHead title={title} description={description} />
+		<BaseHead title={SITE_TITLE} description={SITE_DESCRIPTION} />
 	</head>
-
 	<body>
-		<Header />
-		<div class="wrapper">
-			<main class="content">
-				<section class="intro">
-					<h1>{title}</h1>
-					<p>{description}</p>
-				</section>
-				<section aria-label="Blog post list">
-					{posts.map(({ url, frontmatter }) => (
-						<BlogPostPreview
-							title={frontmatter.title}
-							description={frontmatter.description}
-							publishDate={frontmatter.publishDate}
-							url={url}
-						/>
-					))}
-				</section>
-			</main>
-		</div>
+		<Header title={SITE_TITLE} />
+		<main>
+			<h1>🧑‍🚀 Hello, Astronaut!</h1>
+			<p>
+				Welcome to the official <a href="https://astro.build/">Astro</a> blog starter template. This
+				template serves as a lightweight starting point for anyone looking to build a personal website,
+				blog, or portfolio with Astro.
+			</p>
+			<p>
+				When you are ready, you are free to customize your setup or leave it as it. Add Tailwind for
+				styling (<code>npm run astro add tailwind</code>), or continue to use Astro's built-in
+				scoped styles. Add MDX if you'd like to add components into your Markdown (<code
+					>npm run astro add mdx
+				</code>) or continue to use Astro's built-in plain Markdown support.
```

Comment:
It looks like the blog already uses mdx. Perhaps we can suggest other integrations instead?

Reply:
I'd suggest adding RSS here! (If you don't want to just include this in the template itself... and if you do, update it as a feature in the README with a green checkbox.)

```suggestion
				scoped styles. Add automatic [RSS feed generation](https://docs.astro.build/en/guides/rss/) so others can subscribe to your blog content in a newsreader(<code
					>npm i @astrojs/rss
				</code>) .
```

Reply:
done!